### ISSUE TITLE
Allow Bundle to specify jks keystore alias

### ISSUE THREAD

**Issue by dsand1234** [cert-manager/trust-manager#221]
Right now the alias is not predictable (it seems to be auto generated based on a hash) and my app requires me to specify a keystore alias.  ( The alias should be able to be determined ahead of time without having to deploy the bundle and then extract / examine the keystore that was output.  Ideally, one should be able to specify the alias. )

**Comment by erikgb** [comment]
@dsand1234 Which alias are you referring to here? AFAIK the alias in a _truststore_ should be irrelevant. That is different from a _keystore_, where it matters. You usually trust all root certificates in the _truststore_, so there should be no need to specify an alias. I don't think it will be possible to have predefined aliases, as the root certificates to include in the trust bundle are sourced from PEM. And certificate alias is a keystore/truststore thing. See also https://github.com/cert-manager/trust-manager/issues/44.

**Comment by dsand1234** [comment]
@erikgb  You are right, I was trying to use Bundle to create a keystore rather than a truststore.  What I should be doing is distributing the trust bundle to the various namespaces using Bundle and then having a certificate and issuer in the namespace do the self signing.

**Comment by erikgb** [comment]
You should use cert-manager to issue certificates. Trust-manager is for distributing certificates you trust.
